llvm/test/Instrumentation/BoundsChecking/runtimes.ll

```diff
@@ -5,6 +5,21 @@
 ; RUN: opt < %s -passes='bounds-checking<rt-abort>'     -S | FileCheck %s --check-prefixes=RTABORT
 ; RUN: opt < %s -passes='bounds-checking<min-rt>'       -S | FileCheck %s --check-prefixes=MINRT
 ; RUN: opt < %s -passes='bounds-checking<min-rt-abort>' -S | FileCheck %s --check-prefixes=MINRTABORT
+;
+; merge defaults to true
+; RUN: opt < %s -passes='bounds-checking<merge=true>'              -S | FileCheck %s --check-prefixes=TR
```

Comment:
I think one with trap and one with runtime is enough
we will need later one flag for guard, we don't want 3x them

Reply:
Ok, trimmed the tests